pyproject.toml

```diff
@@ -35,13 +35,11 @@ dependencies = [
     "schedule == 1.1.0",
     "python-multipart >= 0.0.6",
     "openai >= 0.27.8",
```

Comment:
OpenAI yes because it's used for the voice transcription portion.

None of the others, but I don't want to get rid of the database yet, so will deprecate that later.